relay-event-normalization/src/normalize/span/description/mod.rs

```diff
@@ -82,6 +82,26 @@ pub(crate) fn scrub_span_description(span: &Span) -> Option<String> {
                 // They are low-cardinality.
                 Some(description.to_owned())
             }
+            ("contentprovider", "load") => {
```

Comment:
@jjbayer You pointed me to this file initially, is this the only change I need to make to have these ingested internally?

Reply:
You'll also want to extend the `MOBILE_OPS` constant in `defaults.rs`: https://github.com/getsentry/relay/blob/7606bce0845179da2799179aa6cea855d3859b4d/relay-dynamic-config/src/defaults.rs#L20-L21

Reply:
Sounds good, I was thinking about doing that but I was worried adding the entry to `default.rs` would enable it incorrectly for all orgs. I've added them now and also regenerated some snapshots with these new ops

Reply:
That is a good point! I just merged https://github.com/getsentry/relay/pull/2924, so the `Feature::SpanMetricsExtractionAllModules` flag and only extend the mobile ops conditionally.